content/python/concepts/loops/loops.md

```diff
@@ -81,6 +81,30 @@ for group in groups:
     print(name)
 ```
 
+## Do While Loop
```

Comment:
This section should probably come directly after the while loop instead of further down after the nested section